test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
No, I don't mean for it to be aware of jsdoc comments. I'm suggesting that we continue checking indentation in _all_ multiline comments and only allow a single space before the `*` for alignment purposes if the indent rule is configured to use tabs. This kind of alignment applies to all multiline comments, I think.